src/llama_stack/core/datatypes.py

```diff
@@ -387,6 +371,12 @@ class SafetyConfig(BaseModel):
     )
 
 
+class TelemetryConfig(BaseModel):
```

Comment:
hum why is this back? I just read https://github.com/llamastack/llama-stack/pull/4127/files#r2546393372 and though we were removing this? Did I miss something?